src/pkcs7.rs

```diff
@@ -1,20 +1,43 @@
-
 pub mod pkcs7 {
+    use std::fmt;
+    use std::error::Error;
+
+    type Result<T> = std::result::Result<T, Pkcs7UnpadError>;
```

Comment:
Does this provide a benefit other than making the method signature a little simpler? Trying to understand if below should just be specified as `Result<Vec<u8>, Pkcs7UnpadError>`?

Reply:
Honestly, not sure, I was a bit fuzzy on what was the recommended approach to defining results types for rust. I have no objection with using literals though.